### PR TITLE
add X-Ubuntu-Wire-Protocol to the headers

### PR DESCRIPTION
**Pull request by mvo5**
Tiny branch that just sends a wire protocol. I made it a string so that we can have "1.1" versions, however if you feel thats inappropriate I'm happy to convert to a int.

### CODE REVIEW

File: snappy/snapp.go [hunk 1]
```diff
@@ -1757,6 +1762,7 @@ func setUbuntuStoreHeaders(req *http.Request) {
 	req.Header.Set("X-Ubuntu-Frameworks", strings.Join(addCoreFmk(frameworks), ","))
 	req.Header.Set("X-Ubuntu-Architecture", string(Architecture()))
 	req.Header.Set("X-Ubuntu-Release", release.String())
+	req.Header.Set("X-Ubuntu-Core-Wire-Protocol", UbuntuCoreWireProtocol)
```
Comment (niemeyer) [inline snappy/snapp.go]: Sorry, I know this was my own suggestion, but can we please drop the "-Core-" part? Looking here in context and thinking about uses for other future products that will depend on this, I think it's more correct.

With that, LGTM. Thanks!

Comment (mvo5) [inline snappy/snapp.go]: Sure, thats fine!

**Comment by mvo5** [comment]
Thanks, I updated the PR based on the feedback from Gustavo.

**Comment by chipaca** [comment]
:+1:

**Pull request merged**